.rubocop.yml

```diff
@@ -10,134 +8,54 @@ AllCops:
 
 # Lint
 
-# They are idiomatic
-Lint/AssignmentInCondition:
-  Enabled: false
-
-Lint/UnusedMethodArgument:
-  Enabled: false
-
 Lint/ScriptPermission:
+  Enabled: true
   Exclude:
     - 'lib/bundler/templates/Executable'
 
 # Layout
 
-Layout/AccessModifierIndentation:
-  EnforcedStyle: outdent
-
 Layout/AlignParameters:
+  Enabled: true
   EnforcedStyle: with_fixed_indentation
 
-Layout/EndAlignment:
```

Comment:
We shouldn't be removing cops that we have explicitly configured in our code base, I would propose that change in a separate PR/issue. Are these being removed because there are TODOs in `rubocop_todo.yml`?

Reply:
> We shouldn't be removing cops that we have explicitly configured in our code base, I would propose that change in a separate PR/issue.

Well, technically these are not explicitly enabled, only explicitly configured. The only reason they appear here is that their configuration deviates from rubocop's default. The way I see it, they are no different from other cops that were enabled until now, but with the default configuration.

> Are these being removed because there are TODOs in rubocop_todo.yml?

Yes, my strategy here was to remove the TODO file and also remove (instead of explicitly enabling) all cops in `.rubocop.yml` that cause offenses without the TODO file.

That said, the `Layout` cops in particular should be safe to fully enable, so I could do that in a separate PR if you want.

Reply:
Or, I can move the exceptions of these cops to the `.rubocop.yml`, although that's not something I particularly like because if I move the code around without changing its style, rubocop will shout at me because the exception no longer applies (like it happened on https://travis-ci.org/bundler/bundler/jobs/501054430, for example).

Reply:
Could we resolve the TODOs for these rules?

Reply:
As I explained, I don't find these rules more special than others currently enabled implicitly. But I fixed the issues for them anyways in #7014. I left `Layout/AccessModifierIndentation` out, because the auto-correction is crashing and I don't want to do it manually. If don't want to remove this rule, I can move the exception list to the `.rubocop.yml` file.